### ISSUE TITLE
error: the group 'nixbld ' specified in 'build-users-group' does not exist

### ISSUE THREAD

**Issue by sandoor** [NixOS/nix#6064]
**Steps to reproduce**

MacOS Monterey 12.2, zsh. I am reinstalling NIX.
My previous nix install cleanup script

sudo dscl . delete /Groups/nixbld

for x in {1..32}
do
sudo dscl . delete /Users/_nixbld$x
done

#sudo rm -Rfd /nix

if [[ -f  /etc/bashrc.backup-before-nix ]] then
        sudo rm /etc/bashrc
        sudo mv /etc/bashrc.backup-before-nix /etc/bashrc
else
        print "bashrc.backup-before-nix not found"
fi

if [[ -f /etc/bash.bashrc.backup-before-nix ]] then
        sudo rm /etc/bash.bashrc
        sudo mv /etc/bash.bashrc.backup-before-nix /etc/bash.bashrc
else
        print "bash.bashrc.backup-before-nix not found"
fi

if [[ -f /etc/zshrc.backup-before-nix ]] then
        sudo rm /etc/zshrc
        sudo mv /etc/zshrc.backup-before-nix /etc/zshrc
else
        print "zshrc.backup-before-nix not found"
fi

Then I run :

 sh <(curl -L https://nixos.org/nix/install) --darwin-use-unencrypted-nix-store-volume

The script fails at this point :

I am executing:

    $ sudo HOME=/var/root /nix/store/hw5v03wnc0k1pwgiyhblwlxb1fx5zyx8-nix-2.6.0/bin/nix-env -i /nix/store/hw5v03wnc0k1pwgiyhblwlxb1fx5zyx8-nix-2.6.0

to install a bootstrapping Nix in to the default profile

warning: the group 'nixbld ' specified in 'build-users-group' does not exist
installing 'nix-2.6.0'
error: the group 'nixbld ' specified in 'build-users-group' does not exist

If you have a problem with a specific package or NixOS,
you probably want to file an issue at https://github.com/NixOS/nixpkgs/issues. 

So I checked the users and groups.
The stated group nixbld exists.  >> dscl . list /Groups | grep nix returns nixbld

The complete lg follows
% sh <(curl -L https://nixos.org/nix/install) --darwin-use-unencrypted-nix-store-volume
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  4046  100  4046    0     0  10981      0 --:--:-- --:--:-- --:--:-- 10981
downloading Nix 2.6.0 binary tarball for x86_64-darwin from 'https://releases.nixos.org/nix/nix-2.6.0/nix-2.6.0-x86_64-darwin.tar.xz' to '/var/folders/2k/xq1tnqws5tx5f303v1fqlwt80000gn/T/nix-binary-tarball-unpack.XXXXXXXXXX.RQXq1bGy'...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 23.2M  100 23.2M    0     0  21.9M      0  0:00:01  0:00:01 --:--:-- 22.1M
Warning: the flag --darwin-use-unencrypted-nix-store-volume
         is no longer needed and will be removed in the future.

Switching to the Multi-user Installer
Welcome to the Multi-User Nix Installation

This installation tool will set up your computer with the Nix package
manager. This will happen in a few stages:

1. Make sure your computer doesn't already have Nix. If it does, I
   will show you instructions on how to clean up your old install.

2. Show you what I am going to install and where. Then I will ask
   if you are ready to continue.

3. Create the system users and groups that the Nix daemon uses to run
   builds.

4. Perform the basic installation of the Nix files daemon.

5. Configure your shell to import special Nix Profile files, so you
   can use Nix.

6. Start the Nix daemon.

Would you like to see a more detailed list of what I will do?
[y/n] n

---- let's talk about sudo -----------------------------------------------------
This script is going to call sudo a lot. Every time I do, it'll
output exactly what it'll do, and why.

Just like this:

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo echo

to demonstrate how our sudo prompts look

This might look scary, but everything can be undone by running just a
few commands. I used to ask you to confirm each time sudo ran, but it
was too many times. Instead, I'll just ask you this one time:

Can I use sudo?
[y/n] y

Yay! Thanks! Let's get going!

~~> Fixing any leftover Nix volume state
Before I try to install, I'll check for any existing Nix volume config
and ask for your permission to remove it (so that the installer can
start fresh). I'll also ask for permission to fix any issues I spot.

During install, I add 'nix' to /etc/synthetic.conf, which instructs
macOS to create an empty root directory for mounting the Nix volume.
Can I remove /etc/synthetic.conf?
[y/n] y

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo rm /etc/synthetic.conf

to remove /etc/synthetic.conf

During install, I add '/nix' to /etc/fstab so that macOS knows what
mount options to use for the Nix volume.
Can I remove /etc/fstab?
[y/n] y

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo rm /etc/fstab

to remove /etc/fstab

The installer adds a LaunchDaemon to mount your Nix volume: org.nixos.darwin-store
Can I remove it?
[y/n] y

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo launchctl bootout system/org.nixos.darwin-store

to terminate the daemon

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo rm /Library/LaunchDaemons/org.nixos.darwin-store.plist

to remove the daemon definition

~~> Checking for artifacts of previous installs
Before I try to install, I'll check for signs Nix already is or has
been installed on this system.

---- Nix config report ---------------------------------------------------------
        Temp Dir:	/var/folders/2k/xq1tnqws5tx5f303v1fqlwt80000gn/T/tmp.PMPx3nICgQ
        Nix Root:	/nix
     Build Users:	32
  Build Group ID:	30000
Build Group Name:	nixbld

build users:
    Username:	UID
     _nixbld1:	301
     _nixbld2:	302
     _nixbld3:	303
     _nixbld4:	304
     _nixbld5:	305
     _nixbld6:	306
     _nixbld7:	307
     _nixbld8:	308
     _nixbld9:	309
     _nixbld10:	310
     _nixbld11:	311
     _nixbld12:	312
     _nixbld13:	313
     _nixbld14:	314
     _nixbld15:	315
     _nixbld16:	316
     _nixbld17:	317
     _nixbld18:	318
     _nixbld19:	319
     _nixbld20:	320
     _nixbld21:	321
     _nixbld22:	322
     _nixbld23:	323
     _nixbld24:	324
     _nixbld25:	325
     _nixbld26:	326
     _nixbld27:	327
     _nixbld28:	328
     _nixbld29:	329
     _nixbld30:	330
     _nixbld31:	331
     _nixbld32:	332

Ready to continue?
[y/n] y

---- Preparing a Nix volume ----------------------------------------------------
    Nix traditionally stores its data in the root directory /nix, but
    macOS now (starting in 10.15 Catalina) has a read-only root directory.
    To support Nix, I will create a volume and configure macOS to mount it
    at /nix.

~~> Configuring /etc/synthetic.conf to make a mount-point at /nix

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/ex --noplugin /etc/synthetic.conf

to add Nix to /etc/synthetic.conf

~~> Creating a Nix volume

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/diskutil apfs addVolume disk1 APFS Nix Store -nomount

to create a new APFS volume 'Nix Store' on disk1

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/diskutil unmount force disk1s7

to ensure the Nix volume is not mounted

disk1s7 was already unmounted

~~> Configuring /etc/fstab to specify volume mount options

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/vifs

to add nix to fstab

Volume Nix Store on Nix Store mounted

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/security -i

to add your Nix volume's password to Keychain

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/diskutil apfs encryptVolume Nix Store -user disk -stdinpassphrase

to encrypt your Nix volume

Encrypting with the new "Disk" crypto user on disk1s7
The new "Disk" user will be the only one who has initial access to disk1s7
The new APFS crypto user UUID will be 787D92F5-B3B0-49AB-B89E-DEF4FE89F9B3
Encryption has likely completed due to AES hardware; see "diskutil apfs list"
Volume Nix Store on disk1s7 force-unmounted

~~> Configuring LaunchDaemon to mount 'Nix Store'

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/ex --noplugin /Library/LaunchDaemons/org.nixos.darwin-store.plist

to install the Nix volume mounter

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo launchctl bootstrap system /Library/LaunchDaemons/org.nixos.darwin-store.plist

to launch the Nix volume mounter

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo launchctl kickstart -k system/org.nixos.darwin-store

to launch the Nix volume mounter

~~> Setting up the build group nixbld

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o create -r Nix build group for nix-daemon -i 30000 nixbld

Create the Nix build group, nixbld

            Created:	Yes

~~> Setting up the build user _nixbld1

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld1 UniqueID 301

Creating the Nix build user (#1), _nixbld1

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld1 IsHidden 1

in order to make _nixbld1 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld1 NFSHomeDirectory /var/empty

in order to give _nixbld1 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 1

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld1 UserShell /sbin/nologin

in order to give _nixbld1 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld1 nixbld

Add _nixbld1 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld1 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld2

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld2 UniqueID 302

Creating the Nix build user (#2), _nixbld2

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld2 IsHidden 1

in order to make _nixbld2 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld2 NFSHomeDirectory /var/empty

in order to give _nixbld2 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 2

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld2 UserShell /sbin/nologin

in order to give _nixbld2 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld2 nixbld

Add _nixbld2 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld2 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld3

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld3 UniqueID 303

Creating the Nix build user (#3), _nixbld3

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld3 IsHidden 1

in order to make _nixbld3 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld3 NFSHomeDirectory /var/empty

in order to give _nixbld3 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 3

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld3 UserShell /sbin/nologin

in order to give _nixbld3 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld3 nixbld

Add _nixbld3 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld3 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld4

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld4 UniqueID 304

Creating the Nix build user (#4), _nixbld4

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld4 IsHidden 1

in order to make _nixbld4 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld4 NFSHomeDirectory /var/empty

in order to give _nixbld4 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 4

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld4 UserShell /sbin/nologin

in order to give _nixbld4 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld4 nixbld

Add _nixbld4 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld4 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld5

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld5 UniqueID 305

Creating the Nix build user (#5), _nixbld5

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld5 IsHidden 1

in order to make _nixbld5 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld5 NFSHomeDirectory /var/empty

in order to give _nixbld5 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 5

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld5 UserShell /sbin/nologin

in order to give _nixbld5 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld5 nixbld

Add _nixbld5 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld5 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld6

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld6 UniqueID 306

Creating the Nix build user (#6), _nixbld6

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld6 IsHidden 1

in order to make _nixbld6 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld6 NFSHomeDirectory /var/empty

in order to give _nixbld6 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 6

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld6 UserShell /sbin/nologin

in order to give _nixbld6 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld6 nixbld

Add _nixbld6 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld6 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld7

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld7 UniqueID 307

Creating the Nix build user (#7), _nixbld7

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld7 IsHidden 1

in order to make _nixbld7 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld7 NFSHomeDirectory /var/empty

in order to give _nixbld7 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 7

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld7 UserShell /sbin/nologin

in order to give _nixbld7 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld7 nixbld

Add _nixbld7 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld7 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld8

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld8 UniqueID 308

Creating the Nix build user (#8), _nixbld8

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld8 IsHidden 1

in order to make _nixbld8 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld8 NFSHomeDirectory /var/empty

in order to give _nixbld8 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 8

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld8 UserShell /sbin/nologin

in order to give _nixbld8 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld8 nixbld

Add _nixbld8 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld8 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld9

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld9 UniqueID 309

Creating the Nix build user (#9), _nixbld9

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld9 IsHidden 1

in order to make _nixbld9 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld9 NFSHomeDirectory /var/empty

in order to give _nixbld9 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 9

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld9 UserShell /sbin/nologin

in order to give _nixbld9 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld9 nixbld

Add _nixbld9 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld9 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld10

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld10 UniqueID 310

Creating the Nix build user (#10), _nixbld10

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld10 IsHidden 1

in order to make _nixbld10 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld10 NFSHomeDirectory /var/empty

in order to give _nixbld10 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 10

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld10 UserShell /sbin/nologin

in order to give _nixbld10 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld10 nixbld

Add _nixbld10 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld10 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld11

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld11 UniqueID 311

Creating the Nix build user (#11), _nixbld11

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld11 IsHidden 1

in order to make _nixbld11 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld11 NFSHomeDirectory /var/empty

in order to give _nixbld11 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 11

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld11 UserShell /sbin/nologin

in order to give _nixbld11 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld11 nixbld

Add _nixbld11 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld11 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld12

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld12 UniqueID 312

Creating the Nix build user (#12), _nixbld12

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld12 IsHidden 1

in order to make _nixbld12 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld12 NFSHomeDirectory /var/empty

in order to give _nixbld12 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 12

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld12 UserShell /sbin/nologin

in order to give _nixbld12 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld12 nixbld

Add _nixbld12 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld12 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld13

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld13 UniqueID 313

Creating the Nix build user (#13), _nixbld13

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld13 IsHidden 1

in order to make _nixbld13 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld13 NFSHomeDirectory /var/empty

in order to give _nixbld13 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 13

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld13 UserShell /sbin/nologin

in order to give _nixbld13 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld13 nixbld

Add _nixbld13 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld13 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld14

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld14 UniqueID 314

Creating the Nix build user (#14), _nixbld14

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld14 IsHidden 1

in order to make _nixbld14 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld14 NFSHomeDirectory /var/empty

in order to give _nixbld14 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 14

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld14 UserShell /sbin/nologin

in order to give _nixbld14 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld14 nixbld

Add _nixbld14 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld14 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld15

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld15 UniqueID 315

Creating the Nix build user (#15), _nixbld15

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld15 IsHidden 1

in order to make _nixbld15 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld15 NFSHomeDirectory /var/empty

in order to give _nixbld15 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 15

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld15 UserShell /sbin/nologin

in order to give _nixbld15 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld15 nixbld

Add _nixbld15 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld15 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld16

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld16 UniqueID 316

Creating the Nix build user (#16), _nixbld16

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld16 IsHidden 1

in order to make _nixbld16 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld16 NFSHomeDirectory /var/empty

in order to give _nixbld16 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 16

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld16 UserShell /sbin/nologin

in order to give _nixbld16 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld16 nixbld

Add _nixbld16 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld16 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld17

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld17 UniqueID 317

Creating the Nix build user (#17), _nixbld17

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld17 IsHidden 1

in order to make _nixbld17 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld17 NFSHomeDirectory /var/empty

in order to give _nixbld17 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 17

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld17 UserShell /sbin/nologin

in order to give _nixbld17 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld17 nixbld

Add _nixbld17 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld17 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld18

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld18 UniqueID 318

Creating the Nix build user (#18), _nixbld18

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld18 IsHidden 1

in order to make _nixbld18 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld18 NFSHomeDirectory /var/empty

in order to give _nixbld18 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 18

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld18 UserShell /sbin/nologin

in order to give _nixbld18 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld18 nixbld

Add _nixbld18 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld18 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld19

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld19 UniqueID 319

Creating the Nix build user (#19), _nixbld19

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld19 IsHidden 1

in order to make _nixbld19 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld19 NFSHomeDirectory /var/empty

in order to give _nixbld19 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 19

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld19 UserShell /sbin/nologin

in order to give _nixbld19 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld19 nixbld

Add _nixbld19 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld19 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld20

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld20 UniqueID 320

Creating the Nix build user (#20), _nixbld20

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld20 IsHidden 1

in order to make _nixbld20 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld20 NFSHomeDirectory /var/empty

in order to give _nixbld20 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 20

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld20 UserShell /sbin/nologin

in order to give _nixbld20 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld20 nixbld

Add _nixbld20 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld20 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld21

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld21 UniqueID 321

Creating the Nix build user (#21), _nixbld21

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld21 IsHidden 1

in order to make _nixbld21 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld21 NFSHomeDirectory /var/empty

in order to give _nixbld21 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 21

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld21 UserShell /sbin/nologin

in order to give _nixbld21 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld21 nixbld

Add _nixbld21 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld21 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld22

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld22 UniqueID 322

Creating the Nix build user (#22), _nixbld22

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld22 IsHidden 1

in order to make _nixbld22 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld22 NFSHomeDirectory /var/empty

in order to give _nixbld22 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 22

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld22 UserShell /sbin/nologin

in order to give _nixbld22 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld22 nixbld

Add _nixbld22 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld22 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld23

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld23 UniqueID 323

Creating the Nix build user (#23), _nixbld23

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld23 IsHidden 1

in order to make _nixbld23 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld23 NFSHomeDirectory /var/empty

in order to give _nixbld23 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 23

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld23 UserShell /sbin/nologin

in order to give _nixbld23 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld23 nixbld

Add _nixbld23 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld23 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld24

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld24 UniqueID 324

Creating the Nix build user (#24), _nixbld24

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld24 IsHidden 1

in order to make _nixbld24 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld24 NFSHomeDirectory /var/empty

in order to give _nixbld24 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 24

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld24 UserShell /sbin/nologin

in order to give _nixbld24 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld24 nixbld

Add _nixbld24 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld24 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld25

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld25 UniqueID 325

Creating the Nix build user (#25), _nixbld25

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld25 IsHidden 1

in order to make _nixbld25 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld25 NFSHomeDirectory /var/empty

in order to give _nixbld25 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 25

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld25 UserShell /sbin/nologin

in order to give _nixbld25 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld25 nixbld

Add _nixbld25 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld25 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld26

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld26 UniqueID 326

Creating the Nix build user (#26), _nixbld26

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld26 IsHidden 1

in order to make _nixbld26 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld26 NFSHomeDirectory /var/empty

in order to give _nixbld26 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 26

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld26 UserShell /sbin/nologin

in order to give _nixbld26 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld26 nixbld

Add _nixbld26 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld26 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld27

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld27 UniqueID 327

Creating the Nix build user (#27), _nixbld27

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld27 IsHidden 1

in order to make _nixbld27 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld27 NFSHomeDirectory /var/empty

in order to give _nixbld27 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 27

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld27 UserShell /sbin/nologin

in order to give _nixbld27 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld27 nixbld

Add _nixbld27 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld27 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld28

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld28 UniqueID 328

Creating the Nix build user (#28), _nixbld28

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld28 IsHidden 1

in order to make _nixbld28 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld28 NFSHomeDirectory /var/empty

in order to give _nixbld28 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 28

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld28 UserShell /sbin/nologin

in order to give _nixbld28 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld28 nixbld

Add _nixbld28 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld28 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld29

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld29 UniqueID 329

Creating the Nix build user (#29), _nixbld29

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld29 IsHidden 1

in order to make _nixbld29 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld29 NFSHomeDirectory /var/empty

in order to give _nixbld29 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 29

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld29 UserShell /sbin/nologin

in order to give _nixbld29 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld29 nixbld

Add _nixbld29 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld29 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld30

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld30 UniqueID 330

Creating the Nix build user (#30), _nixbld30

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld30 IsHidden 1

in order to make _nixbld30 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld30 NFSHomeDirectory /var/empty

in order to give _nixbld30 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 30

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld30 UserShell /sbin/nologin

in order to give _nixbld30 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld30 nixbld

Add _nixbld30 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld30 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld31

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld31 UniqueID 331

Creating the Nix build user (#31), _nixbld31

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld31 IsHidden 1

in order to make _nixbld31 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld31 NFSHomeDirectory /var/empty

in order to give _nixbld31 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 31

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld31 UserShell /sbin/nologin

in order to give _nixbld31 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld31 nixbld

Add _nixbld31 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld31 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the build user _nixbld32

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . create /Users/_nixbld32 UniqueID 332

Creating the Nix build user (#32), _nixbld32

           Created:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld32 IsHidden 1

in order to make _nixbld32 a hidden user

            Hidden:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld32 NFSHomeDirectory /var/empty

in order to give _nixbld32 a safe home directory

    Home Directory:	/var/empty
              Note:	Nix build user 32

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld32 UserShell /sbin/nologin

in order to give _nixbld32 a safe home directory

   Logins Disabled:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/dseditgroup -o edit -t user -a _nixbld32 nixbld

Add _nixbld32 to the nixbld group

  Member of nixbld:	Yes

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/bin/dscl . -create /Users/_nixbld32 PrimaryGroupID 30000

to let the nix daemon use this user for builds (this might seem redundant, but there are two concepts of group membership)

    PrimaryGroupID:	30000

~~> Setting up the basic directory structure

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /usr/sbin/chown -R root:nixbld /nix

to take root ownership of existing Nix store files

chown: /nix/.Trashes: Operation not permitted
chown: /nix/.Trashes: Operation not permitted

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo install -dv -m 0755 /nix /nix/var /nix/var/log /nix/var/log/nix /nix/var/log/nix/drvs /nix/var/nix /nix/var/nix/db /nix/var/nix/gcroots /nix/var/nix/profiles /nix/var/nix/temproots /nix/var/nix/userpool /nix/var/nix/gcroots/per-user /nix/var/nix/profiles/per-user

to make the basic directory structure of Nix (part 1)

install: mkdir /nix/var
install: mkdir /nix/var/log
install: mkdir /nix/var/log/nix
install: mkdir /nix/var/log/nix/drvs
install: mkdir /nix/var/nix
install: mkdir /nix/var/nix/db
install: mkdir /nix/var/nix/gcroots
install: mkdir /nix/var/nix/profiles
install: mkdir /nix/var/nix/temproots
install: mkdir /nix/var/nix/userpool
install: mkdir /nix/var/nix/gcroots/per-user
install: mkdir /nix/var/nix/profiles/per-user

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo install -dv -g nixbld -m 1775 /nix/store

to make the basic directory structure of Nix (part 2)

install: mkdir /nix/store

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo install -dv -m 0555 /etc/nix

to place the default nix daemon configuration (part 1)

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo install -m 0664 /var/folders/2k/xq1tnqws5tx5f303v1fqlwt80000gn/T/tmp.PMPx3nICgQ/.nix-channels /var/root/.nix-channels

to set up the default system channel (part 1)

~~> Installing Nix

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo cp -RLp ./store/015rgxn5xx5awkz48yhh9c9n1bsvvivm-libiconv-50 ./store/0cgmjzpv8xy2ll3z1wwpnj5js58ddp07-libcxx-7.1.0 ./store/0ma0hl096m764z10k1lp5fwvl7pzm16a-boehm-gc-8.0.4 ./store/0qkfjbhpq6v36rdy8j5arnxdrapv3i3m-libkrb5-1.18 ./store/1j82hjxc28ccvi8i4f3zqg8gia3zvcnp-openssl-1.1.1l ./store/2p623hvx2ha2clb6ry2cyyd0jdz0qyd8-aws-c-cal-0.4.5 ./store/3saax6q1s9im5p80nxhdqpjmw3g4nzg8-curl-7.76.1 ./store/4hdh249mvpywd9qka6awibgdils1mf2a-libcpuid-0.5.1 ./store/5l22hgwshwq7sqn7wqdxmpq1q2100br8-libarchive-3.5.2-lib ./store/5py10pm1w9ziv4qgsk80ghqxwf51syv1-brotli-1.0.9-lib ./store/69dskd4cabzl6cxch5lbfss73l3r6bya-apple-framework-Security ./store/6p1mr7avpnni6n7ssxqjfj7fa4sy00x0-openssl-1.1.1l ./store/79q6vp06aj2p550f2wgn9zywywm6pmir-libcxxabi-7.1.0 ./store/80lzb3092v637di7ihbwx888isy1w1li-libssh2-1.9.0 ./store/8vdyi0ifc098g6cnw38pqbwi78drkgj0-aws-sdk-cpp-1.8.121 ./store/95n0y1033piss1slf99is61a3zc0yv0p-libsodium-1.0.18 ./store/9lzf8irh5b0v9vix6j847vyhqp5lxwq6-aws-checksums-0.1.11 ./store/afpdl5r9jnb8bx42nzpqi0c6g1flwldg-Libsystem-1238.60.2 ./store/cpgyi9khbv1yg2i22vg0zhmsnif9aih9-bash-4.4-p23 ./store/d0dxz2ysp7xd49l8w2n738pps2k3d7m9-aws-c-io-0.9.1 ./store/g8fbhc5bn0mi29gw3jhxm4hrlif00nlz-brotli-1.0.9-lib ./store/gb9nkla1b2zplgbn7j01q7l8hswgwcnb-xz-5.2.5 ./store/hw5v03wnc0k1pwgiyhblwlxb1fx5zyx8-nix-2.6.0 ./store/i0zac1vw0ra6g2mg701m3kc9xbi24lgg-zlib-1.2.11 ./store/i8rksi0pqzcbmsfs48kwp5nvih8mfn4a-bzip2-1.0.6.0.2 ./store/jd1l64vr0b5y1qvdsrminn53gkvamm32-editline-1.17.1 ./store/k5i7ralzb1w0jx3fyn8q9n80kwv16l33-swift-corefoundation ./store/k5vggrdqvyb4lgyy7ck7s7wlyfiqfh9k-sqlite-3.35.5 ./store/k7vg16vz0dsxjfxv0a2hkxw7y4msv7la-zstd-1.4.9 ./store/kldmyqpq68qqp5y80fa0y66zlfzalm7m-aws-c-event-stream-0.2.7 ./store/kvcxphxg6pwp9wgvb468l5m5sbd5krrs-aws-c-common-0.5.5 ./store/kvvpk2vdlh4pc8qyfxcd1rlzyr4zb3bp-libssh2-1.9.0 ./store/n2a3hyr6kmadwpivr0x36xylj9j0wcja-curl-7.76.1 ./store/nqfyfv5lq0n00iqakzcpbl5cq2dkwdz8-libxml2-2.9.12 ./store/r0023xa974x14h83jwrcc8x6i54n9b78-libxml2-2.9.12 ./store/s3dhqrdq6mj935q2pfgw373w0n8459m1-apple-framework-IOKit ./store/wil7lqvyfb18hkdv0bdrf5h0f9y5v9lm-ICU-66108 ./store/wiqcjg66s7sb6cais8pifrk3l9cpkrmq-nss-cacert-3.66 ./store/x0r180c6xm8zgfnczhi27831ly56k888-nghttp2-1.43.0-lib ./store/z31hmk06a669vhjaj6ahn5x2i73l2bsy-libkrb5-1.18 ./store/zqigm4h2qjj8pr6z47aj4zf3rasrb6fy-nghttp2-1.43.0-lib /nix/store/

to copy the basic Nix files to the new store at /nix/store

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo chmod -R ugo-w /nix/store/

to make the new store non-writable at /nix/store

      Alright! We have our first nix at /nix/store/hw5v03wnc0k1pwgiyhblwlxb1fx5zyx8-nix-2.6.0

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo /nix/store/hw5v03wnc0k1pwgiyhblwlxb1fx5zyx8-nix-2.6.0/bin/nix-store --load-db

to load data for the first time in to the Nix Database

warning: the group 'nixbld ' specified in 'build-users-group' does not exist
      Just finished getting the nix database ready.

~~> Setting up shell profiles: /etc/bashrc /etc/profile.d/nix.sh /etc/zshrc /etc/bash.bashrc /etc/zsh/zshrc

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo cp /etc/bashrc /etc/bashrc.backup-before-nix

to back up your current /etc/bashrc to /etc/bashrc.backup-before-nix

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo tee -a /etc/bashrc

extend your /etc/bashrc with nix-daemon settings

# Nix
if [ -e '/nix/var/nix/profiles/default/etc/profile.d/nix-daemon.sh' ]; then
  . '/nix/var/nix/profiles/default/etc/profile.d/nix-daemon.sh'
fi
# End Nix

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo cp /etc/zshrc /etc/zshrc.backup-before-nix

to back up your current /etc/zshrc to /etc/zshrc.backup-before-nix

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo tee -a /etc/zshrc

extend your /etc/zshrc with nix-daemon settings

# Nix
if [ -e '/nix/var/nix/profiles/default/etc/profile.d/nix-daemon.sh' ]; then
  . '/nix/var/nix/profiles/default/etc/profile.d/nix-daemon.sh'
fi
# End Nix

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo cp /etc/bash.bashrc /etc/bash.bashrc.backup-before-nix

to back up your current /etc/bash.bashrc to /etc/bash.bashrc.backup-before-nix

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo tee -a /etc/bash.bashrc

extend your /etc/bash.bashrc with nix-daemon settings

# Nix
if [ -e '/nix/var/nix/profiles/default/etc/profile.d/nix-daemon.sh' ]; then
  . '/nix/var/nix/profiles/default/etc/profile.d/nix-daemon.sh'
fi
# End Nix

~~> Setting up the default profile

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo HOME=/var/root /nix/store/hw5v03wnc0k1pwgiyhblwlxb1fx5zyx8-nix-2.6.0/bin/nix-env -i /nix/store/hw5v03wnc0k1pwgiyhblwlxb1fx5zyx8-nix-2.6.0

to install a bootstrapping Nix in to the default profile

warning: the group 'nixbld ' specified in 'build-users-group' does not exist
installing 'nix-2.6.0'
error: the group 'nixbld ' specified in 'build-users-group' does not exist

---- oh no! --------------------------------------------------------------------
Jeeze, something went wrong. If you can take all the output and open
an issue, we'd love to fix the problem so nobody else has this issue.

:(

We'd love to help if you need it.

You can open an issue at https://github.com/nixos/nix/issues

Or feel free to contact the team:
 - Matrix: #nix:nixos.org
 - IRC: in #nixos on irc.libera.chat
 - twitter: @nixos_org
 - forum: https://discourse.nixos.org

**Comment by abathur** [comment]
1. Why are you reinstalling Nix? Was something already wrong with the install?
2. Please use some markdown formatting around code/log blocks in the future--it's very hard to read.
3. I'm not really sure what's up with the group. There may be some lingering associations between the users and group that your script. It might help to gather some more information, but I'm not terribly familiar with Apple's group settings myself. I'd start with collecting output for:
    - `dscl . list /Users UniqueID | grep nixbld | sort -n -k2`
    - `dscl . read /Users/_nixbld1`
    - `dscl . read /Groups/nixbld`

For reference, here's what these 3 generate on my system (I can't swear it'll be the same--I'm still back on Catalina and my install is a little old):

```console
$ dscl . list /Users UniqueID | grep nixbld | sort -n -k2
_nixbld1                 301
_nixbld2                 302
_nixbld3                 303
_nixbld30                304
_nixbld31                305
_nixbld4                 306
_nixbld5                 307
_nixbld6                 308
_nixbld7                 309
_nixbld8                 310
_nixbld9                 311
_nixbld10                312
_nixbld11                313
_nixbld12                314
_nixbld13                315
_nixbld14                316
_nixbld15                317
_nixbld16                318
_nixbld17                319
_nixbld18                320
_nixbld19                321
_nixbld20                322
_nixbld21                323
_nixbld22                324
_nixbld23                325
_nixbld24                326
_nixbld25                327
_nixbld26                328
_nixbld27                329
_nixbld28                330
_nixbld29                331
_nixbld32                332

$ dscl . read /Users/_nixbld1
dsAttrTypeNative:accountPolicyData:
 <?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>creationTime</key>
	<real>1587337075.454917</real>
	<key>failedLoginCount</key>
	<integer>0</integer>
	<key>failedLoginTimestamp</key>
	<integer>0</integer>
	<key>passwordLastSetTime</key>
	<real>1587337076.0025048</real>
</dict>
</plist>

dsAttrTypeNative:IsHidden: 1
dsAttrTypeNative:record_daemon_version: 6930000
AppleMetaNodeLocation: /Local/Default
GeneratedUID: 2145B9C4-065A-47F5-88EF-8CF81DF14DE8
NFSHomeDirectory: /var/empty
Password: ********
PrimaryGroupID: 30000
RealName: _nixbld1
RecordName: _nixbld1
RecordType: dsRecTypeStandard:Users
UniqueID: 301
UserShell: /sbin/nologin

$ dscl . read /Groups/nixbld
dsAttrTypeNative:record_daemon_version: 6930000
AppleMetaNodeLocation: /Local/Default
GeneratedUID: 9031A2FB-2081-46E3-8D3A-9C3915309943
GroupMembership: _nixbld1 _nixbld2 _nixbld3 _nixbld4 _nixbld5 _nixbld6 _nixbld7 _nixbld8 _nixbld9 _nixbld10 _nixbld11 _nixbld12 _nixbld13 _nixbld14 _nixbld15 _nixbld16 _nixbld17 _nixbld18 _nixbld19 _nixbld20 _nixbld21 _nixbld22 _nixbld23 _nixbld24 _nixbld25 _nixbld26 _nixbld27 _nixbld28 _nixbld29 _nixbld30 _nixbld31 _nixbld32
PrimaryGroupID: 30000
RealName:
 Nix build group for nix-daemon
RecordName: nixbld
RecordType: dsRecTypeStandard:Groups
```

**Comment by abathur** [comment]
FWIW, this also reminds me of another recently opened issue: #6035

It would be good to understand what's causing these, but if you're in a hurry you may also be able to follow the recommendations in that thread.

**Comment by sandoor** [comment]
I have successfully reinstalled NIX, by doing ALL the steps stated in the uninstall instructions. 
What I was not doing previously :

1. Did not sudo rm ALL of the NIX files. Was doing only rm -rf /nix instead of
'''sudo rm -rf /etc/nix /var/root/.nix-profile /var/root/.nix-defexpr /var/root/.nix-channels ~/.nix-profile ~/.nix-defexpr ~/.nix-channels'''

2. Did not "Stop and remove the Nix daemon service"
3. Did not "Remove the Nix Store volume" (did it only once but nothing differed)
4. Did not always restart shell or Reboot...

I was also suspecting the /etc/nix/nix.conf setting build-users-group = nixbld
I might have had <U+2028> at the end of lines (copy pase issue)

Running uninstall script I threw together from your instructions solved the issue.
Had to remove all the hash symbols for posting here because of the formatting.
'''
!/bin/zsh

if [[ -f  /etc/bashrc.backup-before-nix ]] then
        sudo rm /etc/bashrc
        sudo mv /etc/bashrc.backup-before-nix /etc/bashrc
else
        print "bashrc.backup-before-nix not found"
fi

if [[ -f /etc/bash.bashrc.backup-before-nix ]] then
        sudo rm /etc/bash.bashrc
        sudo mv /etc/bash.bashrc.backup-before-nix /etc/bash.bashrc
else
        print "bash.bashrc.backup-before-nix not found"
fi

if [[ -f /etc/zshrc.backup-before-nix ]] then
        sudo rm /etc/zshrc
        sudo mv /etc/zshrc.backup-before-nix /etc/zshrc
else
        print "zshrc.backup-before-nix not found"
fi

sudo nano /etc/synthetic.conf

sudo vifs

sudo launchctl unload /Library/LaunchDaemons/org.nixos.nix-daemon.plist
sudo rm /Library/LaunchDaemons/org.nixos.nix-daemon.plist
sudo launchctl unload /Library/LaunchDaemons/org.nixos.activate-system.plist
sudo rm /Library/LaunchDaemons/org.nixos.activate-system.plist

sudo rm -rf /etc/nix /var/root/.nix-profile /var/root/.nix-defexpr /var/root/.nix-channels ~/.nix-profile ~/.nix-defexpr ~/.nix-channels

sudo dscl . delete /Groups/nixbld

for x in {1..32}
do
sudo dscl . delete /Users/_nixbld$x
done

sudo diskutil apfs deleteVolume /nix

echo "Please reboot for changes to take effect"
'''

Thank you

**Comment by abathur** [comment]
> I was also suspecting the /etc/nix/nix.conf setting build-users-group = nixbld
> I might have had <U+2028> at the end of lines (copy pase issue)

This is probably the root cause. I didn't see it, I don't think, when I looked at this from my laptop last night. But from my phone this morning I see an unrenderable box character appended to nixbld in your log/title. In this case, as long as your system is sorted, you can close the issue?
